example/k3d/environment/main.jsonnet

```diff
@@ -136,7 +136,7 @@ local images = {
 
       local cluster_label = 'k3d-agent/cluster',
       agent_config+: {
-        metrics+: {
+        prometheus+: {
```

Comment:
Shouldn't we switch everything to use `metrics` since prometheus is deprecated?

Reply:
Yeah, I wanted your input on that. It certainly makes more sense, but I wasn't sure if anyone else may be depending on `scraping-svc`, and that change may break their work. WDYT?

Reply:
Nah, I think it's safe to update. It might break people, but the library should've been updated with the name change ages ago and it's arguably a bug that it hasn't been yet :)

Reply:
Okay, I've done that! I'll sanity-test the example once more just to be sure everything works as expected.